guides/common/modules/proc_creating-and-modifying-resources.adoc

```diff
@@ -0,0 +1,37 @@
+[id="creating-and-modifying-resources"]
+= Creating and modifying resources
+
+You can use `curl` to manipulate resources on your {ProjectServer}.
+API calls to {Project} require data in `json` format.
+For more information, see xref:passing-json-data-to-the-api-request[].
+
+[id="api-creating-a-user"]
+.Creating a user
+
+This example creates a user by providing required information in the `--data` option.
+
+Example request:
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ curl --header "Accept:application/json" \
+--header "Content-Type:application/json" --request POST \
+--user _My_User_Name_:__My_Password__ \
+--data "{\"firstname\":\"_Test Name_\",\"mail\":\"_test@example.com_\",\"login\":\"_test_user_\",\"password\":\"_password123_\",\"auth_source_id\":__1__}" \
+https://_{foreman-example-com}_/api/users | python3 -m json.tool
```

Comment:
We currently have a mix of:

````
command | next command
````

and 

````
command \
| next command
````

Does anyone have a source? Or subjective preference?

Reply:
I'd suggest to follow the format initially used in _API request composition_, ie. number two.